feeder/src/main/java/org/n52/sos/importer/feeder/Feeder.java

```diff
@@ -396,6 +408,11 @@ public void importData(final DataFile dataFile)
                 if (!timeSeriesRepository.isEmpty()) {
                     insertTimeSeries(timeSeriesRepository);
                 }
+                if (isUseLastTimestamp
```

Comment:
@MojioMS Synchronize with condition of if statement in line 438. Please extract private boolean method with speaking name that replaces the condition in both statements.